dataset/bin.cpp

```diff
@@ -264,6 +264,23 @@ auto axis_actions(const VariableConstView &var,
   return axes;
 }
 
+auto hide_masked(
```

Comment:
Is `hide_masked` the right name for this? drop_masked better?

Reply:
The point is that it does *not* make a copy of the input with dropped masks. Instead it hides the masked events by setting the bin sizes in question to zero. No copy of the events is made.

Reply:
Yes, I can see that in the implementation. Leave the name as it is.